setup.py

```diff
@@ -93,10 +91,10 @@
     version=version,
     description=__doc__,
     long_description=readme,
-    keywords="invenio_records_lom Invenio",
+    keywords="invenio_records_lom Invenio lom Learning object metadata records",
     license="MIT",
-    author="Graz University of Technology",
-    author_email="info@tugraz.at",
+    author="Mojib Wali",
```

Comment:
Why do you think your name should be here? Normally in the invenio world the invenio group is stated as the author. In our case that would mean that the Graz University of Technology would be the author.